hardhat.config.js

```diff
@@ -36,10 +36,22 @@ const REPORT_GAS = process.env.REPORT_GAS || false
 module.exports = {
     solidity: {
         compilers: [
+            {
+                version: "0.8.19",
+                COMPILER_SETTINGS,
+            },
             {
                 version: "0.8.7",
                 COMPILER_SETTINGS,
             },
+            {
+                version: "0.8.6",
+                COMPILER_SETTINGS,
+            },
+            {
+                version: "0.8.0",
+                COMPILER_SETTINGS,
+            },
             {
                 version: "0.6.6",
                 COMPILER_SETTINGS,
```

Comment:
perhaps we can remove 0.6 and 0.4 now that they're not used? please remove and see if all README commands work?